docs/source/_static/zzz_GENERATED_NOTEBOOK_SOURCE/tertiary/gwas.py

```diff
@@ -295,28 +295,16 @@ def get_sample_info(vcf_df, sample_metadata_df):
 
 # COMMAND ----------
 
-bmiPhenotype = spark.read. \
-                     format("parquet"). \
-                     load(phenotype_path). \
-                     withColumnRenamed("values", "phenotype_values")
-
-# COMMAND ----------
-
-display(bmiPhenotype.select(fx.explode(fx.col("phenotype_values")).alias("bmi")))
+phenotype_df = pd.read_parquet('/dbfs/' + phenotype_path).explode('values').rename({'values': 'bmi'}, axis='columns').reset_index(drop=True)
```

Comment:
It doesn't matter in this specific case as we only have a single trait (BMI), but it would likely be helpful to make this more agnostic to match the common case that there is >1 trait.